js/apps/admin-ui/src/realm-settings/localization/EffectiveMessageBundles.tsx

```diff
@@ -61,15 +66,32 @@ export const EffectiveMessageBundles = ({
   const [activeFilters, setActiveFilters] = useState<
     Partial<EffectiveMessageBundlesSearchForm>
   >({});
-  const themes = serverInfo.themes;
-  const themeKeys = themes
-    ? Object.keys(themes).sort((a, b) => a.localeCompare(b))
-    : [];
-  const themeTypes = Object.values(themes!)
-    .flatMap((theme) => theme.map((item) => item.name))
-    .filter((value, index, self) => self.indexOf(value) === index)
-    .sort((a, b) => a.localeCompare(b));
   const [key, setKey] = useState(0);
+  const themes = serverInfo.themes;
+
+  const themeTypes = useMemo(() => {
+    if (!themes) {
+      return [];
+    }
+
+    return localeSort(
+      Object.keys(themes).filter((key) => key !== "common"),
```

Comment:
Actually, I think I was wrong about this.  Let's remove this filter.  It won't hurt anything.

Reply:
Thanks @ssilvert this is done now.